README.md

```diff
@@ -16,20 +16,6 @@ automatically retried. Any persistent errors can be inspected and are retained i
 
 ## Running locally
 
-For running locally against docker instances of the following services:
-- hmpps-auth
-- hmpps-nomis-prisoner-api
-- visit-scheduler
-- hmpps-nomis-visits-mapping-service
-- localstack
-- run this application independently e.g. in IntelliJ
-
-`docker compose up --scale hmpps-prisoner-from-nomis-migration=0`
-
-or
-
-`docker-compose up  --scale hmpps-prisoner-from-nomis-migration=0`
-
```

Comment:
This no longer works due to localstack being required both inside HMPPS network and required externally